### PR TITLE
Fix ios clear cache

### LINKED ISSUE #741: [BUG] Bug in clearFileCache() on iOS?
I've noticed that my "Documents & Data" Size on my iPhone is getting bigger and bigger, so I looked into it and found around ~1000 *_exif.jpg Files in my AppContainer.
That is weird because I am using only 100 Images at a time and I always do a `.delete()` on iOS after using `.file` or `.orginFile` and on every app start I also do an additional `.clearFileCache()`

I had that problem already and fixed it with `.clearFileCache()` and `.delete()`, but now it seems to be back.
Shouldn't `.clearFileFache()` delete everything cached?
When starting the app I see it disappearing from the iPhone storage Information, so something is happening there, but when it reappears it's still large 

I am using the latest version of flutter_photo_manager (2.0.5, updated today) and iOS (15.4)

### PR DESCRIPTION
**Pull request by CaiJingLong**
closed #741

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,33 +1,45 @@
 # CHANGELOG
 
+## 2.0.6
+
+Fixeds:
```
Comment (AlexV525) [inline CHANGELOG.md:5]:
```suggestion
Fixes:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,33 +1,45 @@
...
+
+- Clear file cache on iOS. (#742)(#743)
```
Comment (AlexV525) [inline CHANGELOG.md:7]:
```suggestion
- Fix file caches clearing on iOS. (#743)
```

File: ios/Classes/core/PMManager.h [hunk 3]
```diff
@@ -16,8 +16,13 @@ typedef void (^ChangeIds)(NSArray<NSString *> *);
 #import "PMResultHandler.h"
 #import "PMConvertProtocol.h"
 
+#define PM_VIDEO_CACHE_PATH @".video"
+#define PM_IMAGE_CACHE_PATH @".image"
+#define PM_FULL_IMAGE_CACHE_PATH @"flutter-images"
```
Comment (AlexV525) [inline ios/Classes/core/PMManager.h:21]: Should we use `pm-images`?

Comment (CaiJingLong) [reply ios/Classes/core/PMManager.h:21]: There is a historical reason here, previously `flutter-images` was used, if it is changed to `pm-images` here, the old images will not be deleted in method.

**Review by AlexV525 — APPROVED** [review]

**Pull request merged**